Pengumpulan tugas 1,3, dan 4

saidatunnizmah opened the pull request:
Ini tugas saya kak

saidatunnizmah committed:
Pengumpulan tugas 1,3, dan 4

farhantaufiks commented:
problem 2 nya gaada,
problem 4 itu optional sih nilai tambah aja, kalau problem 2 nilai utama, harusnya yg utama

saidatunnizmah committed:
Add files via upload

mohon maaf karena kurang teliti dalam membaca soal, ini pengumpulan tugas problem 2

farhantaufiks commented:
lgtm
